lib/SPIRV/SPIRVUtil.cpp

```diff
@@ -848,21 +848,28 @@ ConstantInt *getSizet(Module *M, uint64_t Value) {
 //
 ///////////////////////////////////////////////////////////////////////////////
 int getMDOperandAsInt(MDNode *N, unsigned I) {
-  return mdconst::dyn_extract<ConstantInt>(N->getOperand(I))->getZExtValue();
+  auto *Value = mdconst::dyn_extract_or_null<ConstantInt>(N->getOperand(I));
```

Comment:
```suggestion
  if (auto *Value = mdconst::dyn_extract_or_null<ConstantInt>(N->getOperand(I)))
    return Value->getZExtValue();
  return 0;
```

Reply:
Do I understand correctly that this aims at making the code more explicit? (i.e. I'm totally down with this reasoning)
What concerns me most about this particular function is whether it's a good idea to return 0 in case the `MDOperand` is not an integer value, I'll see if there's a way to avoid this and still fulfil my purpose in `LLVMLoopParallelAccessIndices::initialize()`

Reply:
I've decided to restore the original version of the function and do the required "custom" extraction of the safelen value directly.